scripts/apps/vocabularies/controllers/VocabularyEditController.tsx

```diff
@@ -175,7 +255,7 @@ export function VocabularyEditController(
             return false;
         }
 
-        if ($scope.vocabulary.field_options == null || $scope.vocabulary.field_options.allowed_types == null) {
+        if ($scope.vocabulary.field_type !== 'related_content') {
```

Comment:
Are you sure this change is correct? It might be - just looks suspicious to me.

Reply:
It's correct because the type that was used for checking has these fieldOptions